### PR TITLE
Adds helper card in cms page

### PR DESCRIPTION
**Pull request by tomas862**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Adds helper card in cms page located in **/admin-dev/index.php/improve/design/cms-pages**
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | Check the page how it looks like - also when everything looks okey, try to click on 'x' to remove that block. Before the testing make sure assets are up to date by running `npm run build`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13163)

### CODE REVIEW

**Comment by matks** [comment]
@tomas862 Do you feel confident about the RTL behavior :trollface: ?

**Comment by tomas862** [comment]
haha, no :D but I guess existing  showcase cards had this thing handled? :thinking:

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ *#}
+
+{% block cms_pages_showcase_card %}
+  {% if not showcaseCardIsClosed %}
+    <div id="cms-pages-showcase-card" class="showcase-card card">
+      <div class="showcase-card__left shape-one helper-card__cms-pages-background">
+        <img src="{{ asset('themes/new-theme/scss/img/helper-card/content@3x.png') }}" class="img-fluid">
+      </div>
+      <div class="showcase-card__right">
+        <h2>{{ 'Create meaningful content'|trans({}, 'Admin.Design.Help') }}</h2>
```
Comment (tomas862) [inline src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig:33]: Hi @LouiseBonnard ,

does the translation domain looks good here?

**Comment by tomas862** [comment]
> @tomas862 Do you feel confident about the RTL behavior :trollface: ?

close enough:

Screenshot(12)

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig (hunk 1 above)
Comment (LouiseBonnard) [reply src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig:33]: It is perfect, thank you.

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <p>{{ 'PrestaShop enables you to create content pages as easily as you would create product pages. You can add static pages in the top menu using the "Main menu" module.'|trans({}, 'Admin.Design.Help') }}</p>
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig]: It seems that, for my part, I don't have the same wording:

_Because it is not just selling products but also creating a universe, build pages to tell stories and catch your visitors’ interest to turn them into loyal customers._

@TristanLDD, do you know what is the final wording?

**Comment by matks** [comment]
> haha, no :D but I guess existing showcase cards had this thing handled? 🤔

I checked, you need to add CSS class `img-rtl`

See https://github.com/PrestaShop/PrestaShop/pull/12809/files

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig (hunk 2 above)
Comment (matks) [reply src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig]: According to https://github.com/PrestaShop/PrestaShop/issues/10434#issuecomment-481319058 the wording of Louise is the right one @tomas862 can you please update it ?

Comment (tomas862) [reply src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig]: @matks done ;)

**Comment by matks** [comment]
@tomas862 I'm sorry but your commits history looks bad :/ I think you might have used `develop` as base branch instead of 1.7.6.x

**Comment by tomas862** [comment]
snap - old habits of rebasing... :/ gonna cherry pick this PR and I'll create new one instead

**Comment by tomas862** [comment]
Forwarded to https://github.com/PrestaShop/PrestaShop/pull/13404

**Pull request closed without merging**